plugins/woocommerce/client/legacy/js/frontend/address-i18n.js

```diff
@@ -17,7 +17,10 @@ jQuery( function( $ ) {
 				field.find( 'label' ).append(
 					'&nbsp;<abbr class="required" title="' +
 					wc_address_i18n_params.i18n_required_text +
-					'">*</abbr>'
+					'" aria-hidden="true">*</abbr>' + 
```

Comment:
Prevent screen readers from announcing the asterisk and make them announce the term "required" instead.